client/views/HistoryView.vue

```diff
@@ -52,6 +53,12 @@
       }
     },
 
+    watch: {
```

Comment:
Great job! :fireworks: 
Can we add an `initialized` property on the store to watch instead of `oodler`? This would make the code more obvious, especially if something like this is used in other components.